src/main/java/org/fcrepo/auth/webac/WebACAuthorizationDelegate.java

```diff
@@ -0,0 +1,38 @@
+package org.fcrepo.auth.webac;
+
+import java.util.Set;
+
+import javax.jcr.Session;
+
+import org.fcrepo.auth.roles.common.AbstractRolesAuthorizationDelegate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Authorization Delegate responsible for resolving Fedora's permissions using Web Access Control (WebAC) access
+ * control lists.
+ *
+ * @author Peter Eichman
+ * @date Aug 24, 2015
```

Comment:
@awoods I was following the [F4 code style guide](https://wiki.duraspace.org/display/FF/Code+Style+Guide), which does mention a `@date` Javadoc tag. I assume that the style guide should be updated too, and change `@date` to `@since`?